src/boards_config.h

```diff
@@ -3,8 +3,10 @@
 const pins_t pins_v0_5 = {
     .pusherDriverType = HBRIDGE_DRIVER,
     .revSwitch = 15,
-    .triggerSwitch = 33,
-    .cycleSwitch = 25,
+    .triggerSwitch = 32,
+    .cycleSwitch = 23,
+    .select1 = 25,
```

Comment:
This is probably not the right way to handle specific blasters. @ahalekelly we need to think about how we want to support UX options, though it should probably live in `Configuration.h`

Reply:
Yeah true, maybe we should move the switch pin config from boards_config.h to configuration.h? We can do that in a second commit though